package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "cast-and-crew",
+  "version": "1.0.0",
+  "description": "A module for displaying the works of a selected cast/crew member.",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest",
+    "server-dev": "nodemon server/index.js",
+    "react-dev": "webpack -d --watch",
+    "seed": "node -e 'require(\"./database/seeder.js\").seed()'"
```

Comment:
Just have the `seed()` function run automatically so you can just run `node ./database/seeder`.